back-end/sports_platform/sports_platform_api/views/user_views.py

```diff
@@ -1,15 +1,36 @@
 from django.db.utils import IntegrityError
+from rest_framework.authtoken.models import Token
 from rest_framework.decorators import api_view
 from rest_framework.response import Response
-from ..controllers.guest import Guest
-from ..validation import user_validation
-from django.contrib.auth.decorators import login_required
-from ..models import User
+
 from ..controllers import Guest
-import re
-from rest_framework.authtoken.models import Token
-import django.contrib.auth 
+from ..models import User
+from ..serializers.user_serializer import UserSerializer
+from ..validation import user_validation
+
 
+@api_view(['GET'])
+def get_user(request, user_id):
+    if not request.user.is_authenticated:
+        return Response(data={'message': 'User is not logged in, first you need to login'}, status=401)
```

Comment:
Should the user be logged in to be viewed by another user? Maybe we can ask this to other teams.

Reply:
I do not understand what you mean. Users should **not** be logged in to be viewed by another user but they should be **logged** **in** to view other users.
I mean, for this endpoint we have 2 users:
1. User that wants to see another user. This must be logged in
2. User that will be shown. This does not have to be logged in.

Is this what you understood?

Reply:
I am talking about the first case. For example I can still see the profile page of an Instagram user even when I am not logged in to Instagram. So maybe we can also provide this to not logged-in people?

Reply:
I have changed it such that login is not required anymore.